src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -95,6 +96,10 @@ export const BaseCourseSearchResults = (props) => {
       accessor: 'advertised_course_run.start',
       Cell: FormattedDateCell,
     },
+    {
+      Header: TABLE_HEADERS.partnerName,
+      accessor: 'partners[0].name',
```

Comment:
I'm struggling to find it in admin portal. Where does the data for this actually come from? I assume an API call to something, but I'm unclear on what. I'd just like to verify the data format here.

Reply:
ah it's in the response of algolia. So it looks like this in devtools... Also if you look directly in algolia, this format also exists there in the catalog data

```
partners: 
  0: {name: "edX",…}
```

Reply:
Awesome. Thank you